docs/kernel-policy.md

```diff
@@ -21,22 +21,33 @@ and removed in a following release, after its minimum end of support date.
 
 ### Host Kernel
 
-| Host kernel | Min. version | Min. end of support |
-| ----------: | -----------: | ------------------: |
-|       v5.10 |       v1.0.0 |          2024-01-31 |
-|        v6.1 |       v1.5.0 |          2025-10-12 |
+| Host kernel | Min. version | Min. end of support | | ----------: |
+-----------: | ------------------: | | v5.10 | v1.0.0 | 2024-01-31 | | v6.1 |
+v1.5.0 | 2025-10-12 |
```

Comment:
These are looking broken. Seems you missed `\n`.

Reply:
sorry about that, that's my mdformatter changing things without me asking it to. Will fix